### ISSUE TITLE
Clarification on cv_selection_metric

### ISSUE THREAD

**Issue by ggerogiokas** [linkedin/greykite#49]
Hi all,

Thanks for the great work.  Just had a quick question about the cv_selection_metric.
When you choose it does it actually use the selected metric as a loss function or 
is MSE still used and metrics later computed on the CV splits?

Best,
George

**Comment by KaixuYang** [comment]
Hi @ggerogiokas the loss function depends on the fit algorithm and its parameters you choose. For example, if you fit ridge regression, the loss function will be MSE (default loss function in `sklearn.linear_model.ridgeCV`). If you fit a tree model, you may be able to use quantile loss or so. These are independent of the `cv_selection_metric`.  The CV computes metrics on the fitted results and choose the best model according to the `cv_selection_metric`.
